owasp-top10-2017-apps/a6/stegonography/app/index.js

```diff
@@ -85,11 +85,11 @@ router.post("/login", function(req,res)  {
     };
 
     VerifiesUser((username) => { 
-        if (username == "admin") {
+        if (username == "Dino01") {
```

Comment:
👀 🧐

Reply:
🔥This is Fine.🔥👀 
A brand new commit arrives!